LayoutTests/requestidlecallback/requestidlecallback-deadline-shortened-by-rendering-update.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html><!-- webkit-test-runner [ RequestIdleCallbackEnabled=true ] -->
```

Comment:
Why do we need `RequestIdleCallbackEnabled=true` ? Isn't this feature getting enabled by default during tests, like all experimental features?

Reply:
No, it's a not experimental feature yet since the implementation was really broken.